manifests/art.yaml

```diff
@@ -7,8 +7,8 @@ updates:
     # replace entire version line, otherwise would replace {MAJOR}.{MINOR}.0 anywhere
     - search: "version: {MAJOR}.{MINOR}.0"
       replace: "version: {FULL_VER}"
-    - search: 'olm.skipRange: ">=4.4.0 <{MAJOR}.{MINOR}.0"'
-      replace: 'olm.skipRange: ">=4.4.0 <{FULL_VER}"'
+    - search: 'olm.skipRange: ">=4.3.0 <{MAJOR}.{MINOR}.0"'
+      replace: 'olm.skipRange: ">=4.3.0 <{FULL_VER}"'
```

Comment:
https://github.com/openshift/cluster-kube-descheduler-operator/blob/release-4.6/manifests/4.6/cluster-kube-descheduler-operator.v4.6.0.clusterserviceversion.yaml#L26 needs to be updated to `>=4.3.0 <4.6.0` as well.